modules/apps/transfer/keeper/migrations.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v4/modules/apps/transfer/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// This migration will update the denom
+func (m Migrator) MigrateTraces(ctx sdk.Context) error {
```

Comment:
nit: godocs should start with function name. e.g. "MigrateTraces will update the denom traces stored in state"